config.go

```diff
@@ -24,7 +24,7 @@ var (
 	BuildRevision = ""
 
 	// Commandline Version String; used to control auto-updating.
-	ClientVersion = "2019-01-22"
+	ClientVersion = "2019-02-19"
```

Comment:
The validator is on the server, so you don't actually need to change the client version

Reply:
Ah, sorry, that's my fault.

Reply:
I undid this change.